### PR TITLE
Update app.config.yml

### PR DESCRIPTION
**Pull request by brscag10**

### CODE REVIEW

File: app.config.yml [hunk 1]
```diff
@@ -84,3 +86,4 @@ slots:
       - "tile placement"
       - "time limits"
       - "bartering"
+      - "king making"
```
Comment (daniellacosse) [inline app.config.yml]: is kingmaking one word?

Comment (brscag10) [reply app.config.yml]: kingmaking

**Comment by daniellacosse** [comment]
fyi @brscag10 this looks good something else is broken - this is on my list don't you worry

**Comment by brscag10** [comment]
Love uuuu

**Pull request merged**